src/OrchardCore.Modules/OrchardCore.Workflows/Activities/CorrelateTask.cs

```diff
@@ -28,14 +30,21 @@ public WorkflowExpression<string> Value
             set => SetProperty(value);
         }
 
+        public bool IsLiquidSyntax
+        {
+            get => GetProperty(() => false);
+            set => SetProperty(value);
+        }
```

Comment:
Let's use a string instead, so the syntax config is a bit more flexible. Most possibly never needed, but the general experience with such `bool IsOneOfTheThings` values is that they tend to get a third option with time too, and then it's messy to change.

That said... `IWorkflowExpressionEvaluator` is about a generic expression, and while its currently only (non-test) implementation is `LiquidWorkflowExpressionEvaluator`, this doesn't need to be the case. So maybe rather `IsExpressionSyntax` and change the rest of the UI wording too? Hmm, but then we can't have proper syntax highlighting.

Yeah, let's go with a string, and if we were to have multiple expression engines then we can revisit.

Reply:
done